templates/fine-tune-GPTJ/README.md

```diff
@@ -0,0 +1,62 @@
+
+The lines of code to update look like the following:
+![Code](https://github.com/anyscale/templates/releases/download/media/code.png)
+
+Once the code is updated, run the same command as before:
+```
+anyscale job submit -- python gptj_deepspeed_fine_tuning.py
+```
+
+## Saving your model
+The fine tuning job automatically saves checkpoints during training in your [default mounted user storage](https://docs.anyscale.com/develop/workspaces/storage#user-storage).  You can view the model by navigating to "Files" viewer and selecting "User Storage".
+![Files](https://github.com/anyscale/templates/releases/download/media/files.png)
+
+Within 2 minutes you will be fine-tuning GPT-J on a corpus of Shakspeare data!  Let's dive in and explore the power of Anyscale and Ray together.
+
+
+## Appendix
+
+### Advanced - Workspaces and Configurations
+This application makes use of [Anyscale Workspaces](https://docs.anyscale.com/develop/workspaces/get-started) and Ray AIR (with the 🤗 Transformers integration) to fine-tune an LLM. Workspace is a fully managed development environment focused on developer productivity. With workspaces, ML practitioners and ML platform developers can quickly build distributed Ray applications and advance from research to development to production easily, all within single environment.
+
+To run this example, we've set up your Anyscale Workspace to have access to a head node with one GPU with 16 or more GBs of memory and 15 g4dn.4xlarge instances for the worker node group. This is done by defining a "compute configuration".  Learn more about [Compute Configs here] (https://docs.anyscale.com/configure/compute-configs/overview).  It is easy to change your Compute Config once you launch by clicking "Workspace" and Editing the selection.  
+![Config](https://github.com/anyscale/templates/releases/download/media/edit.png)
+
+
+When you run the fine tuning job we execute a python script thats distributed with Ray as an [Anyscale Job](https://docs.anyscale.com/productionize/jobs/get-started).   
+
+### Advanced: Build off of this template's cluster environment
+#### Option 1: Build a new cluster environment on Anyscale
+You'll find a cluster_env.yaml file in the working directory of the template. Feel free to modify this to include more requirements, then follow [this](https://docs.anyscale.com/configure/dependency-management/cluster-environments#creating-a-cluster-environment) guide to use the Anyscale CLI to create a new cluster environment.
+
+Finally, update your workspace's cluster environment to this new one after it's done building.
+
+#### Option 2: Build a new docker image with your own infrastructure
+Use the following docker pull command if you want to manually build a new Docker image based off of this one.
+
+```bash
+docker pull us-docker.pkg.dev/anyscale-workspace-templates/workspace-templates/fine-tune-gptj:latest
```

Comment:
I think the latest tag needs to be manually added for google artifact registry, it's a bit annoying..